src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -178,37 +185,72 @@ public void testDoPost_StoresMessage() throws IOException, ServletException {
     Mockito.verify(mockMessageStore).addMessage(messageArgumentCaptor.capture());
     Assert.assertEquals("Test message.", messageArgumentCaptor.getValue().getContent());
 
+      ArgumentCaptor<Activity> activityArgumentCaptor = ArgumentCaptor.forClass(Activity.class);
```

Comment:
The ArgumentCaptor does exactly that: it captures the passed argument of the given class when it is passed to a function we called `.capture()` on